store/sqlstore/upgrade.go

```diff
@@ -915,3 +917,13 @@ func precheckMigrationToVersion528(sqlStore SqlStore) error {
 
 	return nil
 }
+
+func upgradeDatabaseToVersion529(sqlStore SqlStore) {
+	// if shouldPerformUpgrade(sqlStore, VERSION_5_28_0, VERSION_5_29_0) {
+
+	sqlStore.AlterColumnTypeIfExists("SidebarCategories", "Id", "VARCHAR(255)", "VARCHAR(128)")
```

Comment:
The statements may not be necessary since these actually match the types of the fields in my local DB, but I figure it's good to confirm their types in case some DB versions behave differently.